static/elements/chromedash-upcoming.js

```diff
@@ -52,12 +52,110 @@ class ChromedashUpcoming extends LitElement {
       loginUrl: {type: String},
       starredFeatures: {type: Object}, // will contain a set of starred features
       cardWidth: {type: Number},
+      lastFetchedOn: {type: Number},
+      lastMilestoneVisible: {type: Number},
+      milestoneArray: {type: Array},
+      milestoneInfo: {type: Object},
+      cardsToFetchInAdvance: {type: Number},
```

Comment:
In fact, you could write a source code comment for each property declaration.

cardsToFetchInAdvance seems like it could be more clear as a JS const, unless there would ever be a reason to change it while the user is using the page.

Reply:
Done. See [this](https://github.com/GoogleChrome/chromium-dashboard/pull/1458/commits/4cc0df65ad033941914fe8f9250e23d90f250158) commit